content/guides/embed/ui-elements/explorer-metadata.md

```diff
@@ -0,0 +1,245 @@
+  - embed/ui-elements/installation
+related_resources: []
+alias_paths:
+  - /docs/box-content-explorer
+  - /docs/content-explorer
+---
+
+# Content Explorer - metadata view
+
+With Content Explorer you can also display files and folders
+based on their metadata.
+The metadata view uses [metadata template][template] and [metadata query][metadata-query] to
+find the data you want to display.
+
+## Prerequisites
+
+* Read the [Content Explorer][explorer] guide.
+* Check the [metadata terminology][terminology].
+* Check the information on [metadata queries][metadata-query].
+
+## Create and configure an app
+
+1. [Create a Box app][box-app].
+2. Add the local development address in the CORS Domains: ![CORS Domains](./images/box-app-cors.png)
+3. Generate a [developer token][token].
+
+## Create a metadata template
+
+The next step is to create a metadata template you will use to populate the Content Explorer.
+
+1. Create a metadata template. You can use [Metadata API][creating-templates-api] or [Admin Console][creating-templates-ui] to do so.
+2. Apply an already created template to a Box folder. Make sure you enable the
+cascade policy. For detailed instructions, see
+[instructions on customizing and applying templates][apply-templates].
+
+### Display name and key parameters
+
+* The `displayName` parameter is the display name of the template visible
+in the Admin Console.
+* The `templateKey` parameter is a unique identifier of the template. It needs 
+to be unique across the enterprise for which you create the metadata template.
+If you don't provide the `templateKey` parameter, API creates a unique one
+based on the value in `displayName`.
+* The `[fields].displayName` parameter is the display name of the field as it
+is shown to the user in the web and mobile apps.
+* The `[fields].key` parameter is a unique identifier for a specific field in
+the template. The identifier must be unique within the template to which it
+belongs.
+
+## Display metadata view
+
+To make things easier, you can use a [sample project][metadata-project] to
+launch metadata view.
+
+1. Clone the metadata sample project.
+2. Update the placeholders in [`App.js`][appjs] with actual values:
+
+| Parameter | Description |
+| --- | --- |
+| `DEVELOPER_TOKEN` | [Developer token][token] generated in the the Developer Console. |
+| `ENTERPRISE_ID` | Enterprise ID copied from the **General Settings** tab of your application. |
+| `METADATA_TEMPLATE_NAME`| Name of your already created metadata template. **Note**: To make sure you provided the proper name, use the [metadata API][get-template] to retrieve the name, or copy it from the URL in the Admin Console. ![Metadata name in Admin Console](./images/metadata-template-name.png) If you decide to change the template name in the UI, you change the label only. The name to use in the component is always the you provided at the beginning. |
+| `ROOTFOLDER_ID` | ID of Box folder to which you applied the metadata template. |
+
```

Comment:
Can we add metadata source? and link to the guide about the metadata scoping? 
https://developer.box.com/guides/metadata/scopes/